assets/sass/v2/_okta-verify.scss

```diff
@@ -24,6 +24,11 @@
   .switch-channel-link {
     color: $link-text-color;
   }
+
+  .o-form-fieldset-container + .resend-ov-link-view {
```

Comment:
https://github.com/okta/okta-signin-widget/pull/1762/files#diff-813d68faf1596f458076d3051347512d557042a3098d7e735c7db2989b424c89R87

resend is currently appended to error container not field set container.